### ISSUE TITLE
Delay increase random seconds?

### ISSUE THREAD

**Issue by HelplessMan** [AutomaApp/automa#626]
Can hi add random seconds to the delay module? This function is also supported by the number of seconds between retries of some modules

**Comment by Kholid060** [comment]
Yes, you can add a random to the delay block by using the [`$randint()`](https://docs.automa.site/api-reference/reference-data.html#randint-min-max) function. For example, `{{$randint(1000, 15000)}}`, this will generate a random number between 1000 to 15000.
![image](https://user-images.githubusercontent.com/22908993/176125485-c74e3599-07d6-4233-80dd-d7641fb2d91c.png)

**Comment by HelplessMan** [comment]
> Yes, you can add a random to the delay block by using the [`$randint()`](https://docs.automa.site/api-reference/reference-data.html#randint-min-max) function. For example, `{{$randint(1000, 15000)}}`, this will generate a random number between 1000 to 15000. ![image](https://user-images.githubusercontent.com/22908993/176125485-c74e3599-07d6-4233-80dd-d7641fb2d91c.png)

Thank You!!!!

**Comment by HelplessMan** [comment]
Another question for help..

I want to generate a random string at the beginning of the run.
![image](https://user-images.githubusercontent.com/9101275/176139986-ed6c3fd2-052b-44aa-b1e6-7ab165b6fa1b.png)

Then, when processing the download, the file name is modified to the current random string, and the downloaded file is passed to the variable

![image](https://user-images.githubusercontent.com/9101275/176140251-c9b6354b-6292-44d7-abf6-2cce17917fdf.png)

Then upload the file with variable values to the specified element

![image](https://user-images.githubusercontent.com/9101275/176140611-3b2ba658-2d4d-4ebb-8897-afddf3b7a9b3.png)

Is there anything wrong with my process? Now run time? The actual file name has not changed......

**Comment by HelplessMan** [comment]
> Another question for help..
> 
> I want to generate a random string at the beginning of the run. ![image](https://user-images.githubusercontent.com/9101275/176139986-ed6c3fd2-052b-44aa-b1e6-7ab165b6fa1b.png)
> 
> Then, when processing the download, the file name is modified to the current random string, and the downloaded file is passed to the variable
> 
> ![image](https://user-images.githubusercontent.com/9101275/176140251-c9b6354b-6292-44d7-abf6-2cce17917fdf.png)
> 
> Then upload the file with variable values to the specified element
> 
> ![image](https://user-images.githubusercontent.com/9101275/176140611-3b2ba658-2d4d-4ebb-8897-afddf3b7a9b3.png)
> 
> Is there anything wrong with my process? Now run time? The actual file name has not changed......
